x-pack/plugins/cases/public/components/cases_context/cases_context_reducer.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
This reducer will be only to handle the state for the CasesContext which will handle the flyout, modals and in general third integrations.